src/global/providers/db.provider.ts

```diff
@@ -18,6 +18,9 @@ export const DbProvider: FactoryProvider = {
       user: dbConfig.user,
       password: dbConfig.password,
       database: dbConfig.name,
+      ssl: {
+        rejectUnauthorized: false,
```

Comment:
If memory serves, we use also this provider for production configurations and in those environments we'll want this to be true. This is likely a small change to something like ```rejectUnauthorized: process.env.NODE_ENV === "production"```

Reply:
(per offline chat and likely a massive oversimplification of the details)

this change is needed locally because we're using http and not https

a deployed API server shouldn't require these changes because it'd have an issued certificate

Reply:
@TangoYankee see [014aa8d](https://github.com/NYCPlanning/ae-zoning-api/pull/198/commits/014aa8d601a3e6b5d59b4cae8cf7d53e136a9a19)